.github/workflows/verify-markdown-formatting.yml

```diff
@@ -16,4 +16,14 @@ jobs:
         uses: actions/checkout@v2
 
       - name: "Verify Markdown formatting"
-        run: npx prettier@2.0.4 --check "**/*.md"
+        run: |
+          if [ "$GITHUB_REF" == "refs/heads/master" ]; then
+            echo "Checking the formatting of all Markdown files in this repository"
+            npx prettier@2.0.4 --check "**/*.md"
+          else
```

Comment:
I think it might be nicer if we split this into two conditional steps on the workflow level: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif

Reply:
Done! Much nicer indeed.